Products/ZenModel/Organizer.py

```diff
@@ -355,8 +360,12 @@ def getOrganizer(self, path):
         >>> dmd.Events.Status.getOrganizer('/Events/Status/Snmp')
         <EventClass at /zport/dmd/Events/Status/Snmp>
         """
-        if path.startswith("/"): path = path[1:]
-        return self.getDmdRoot(self.dmdRootName).unrestrictedTraverse(path)
+        if path.startswith("/"):
```

Comment:
path = path.lstrip('/')
path = "/".join(prep(s) for s in path.split("/"))
unrestrictedTraverse